src/app/tests/suites/certification/Test_TC_LVL_3_1.yaml

```diff
@@ -23,117 +23,110 @@ tests:
       cluster: "DelayCommands"
       command: "WaitForCommissionee"
 
-    - label: "reads current level attribute from DUT"
+    - label: "reads current Level attribute from DUT"
       command: "readAttribute"
-      attribute: "current level"
+      attribute: "current Level"
```

Comment:
I don't understand.  In other places in this PR, the yaml was switched from "current Level" to "current level", but here it's being switched in the opposite direction?

Reply:
i will update it